test/unit/phys/domains/cluster/CMakeLists.txt

```diff
@@ -4,9 +4,9 @@ dca_add_gtest(cluster_operations_test
   GTEST_MAIN
   LIBS ${LAPACK_LIBRARIES} ${DCA_CUDA_LIBS})
 
-dca_add_gtest(momentum_exchage_domain_test
-  GTEST_MAIN
-  LIBS cluster_domains)
+#dca_add_gtest(momentum_exchage_domain_test
```

Comment:
Why is this test commented?

Reply:
I changed heavily the requirements from the Parameters, did not have time to update, and forgot about it.
Test re-enabled in 5c52df1153a1a7589a63d09953374793bccfa39b.